cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterAutoCreateTopicTest.java

```diff
@@ -95,11 +100,11 @@ public Properties overridingProps() {
         props.setProperty(CruiseControlMetricsReporterConfig.CRUISE_CONTROL_METRICS_TOPIC_NUM_PARTITIONS_CONFIG, "1");
         props.setProperty(CruiseControlMetricsReporterConfig.CRUISE_CONTROL_METRICS_TOPIC_REPLICATION_FACTOR_CONFIG, "1");
         // disable topic auto-creation to leave the metrics reporter to create the metrics topic
-        props.setProperty(ServerLogConfigs.AUTO_CREATE_TOPICS_ENABLE_CONFIG, "false");
-        props.setProperty(ServerLogConfigs.LOG_FLUSH_INTERVAL_MESSAGES_CONFIG, "1");
-        props.setProperty(GroupCoordinatorConfig.OFFSETS_TOPIC_REPLICATION_FACTOR_CONFIG, "1");
-        props.setProperty(ReplicationConfigs.DEFAULT_REPLICATION_FACTOR_CONFIG, "2");
-        props.setProperty(ServerLogConfigs.NUM_PARTITIONS_CONFIG, "2");
+        props.setProperty("auto.create.topics.enable", "false");
+        props.setProperty("flush.messages", "1");
```

Comment:
Why is this needed? This is usually not recommended with Kafka.

Reply:
I accidentally translated this to the incorrect property, it is supposed to be `log.flush.interval.messages` as it is in the `CruiseControlMetricsReporterTest` and `CruiseControlMetricsReporterSslTest` classes! 

That being said, I am not sure how necessary its inclusion is, the tests seem to run fine without it. I left the property in since it was already part of these tests. This change here is just for removing dependence on the internal Kafka API, `org.apache.kafka.coordinator.group.GroupCoordinatorConfig`. 

If you are confident that the property is unnecessary, I will remove it (The tests pass without it anyway)

Reply:
Yeah, I don't see how this can be related to this test, so let's remove it.

Reply:
Just removed!